src/offset_of.rs

```diff
@@ -350,7 +382,7 @@ mod tests {
 
     #[cfg(any(
         feature = "unstable_const",
-        feature = "unstable_offset_of",
+        any(feature = "unstable_offset_of", stable_offset_of),
```

Comment:
Nested `any` doesn't make a lot of sense, this can be flattened into a single `any`

Reply:
Oh, yeah lol. I just did a find/replace.